Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM golang:1.12.5-alpine3.9 as build
+
+# git is not included, weirdly, so nothing with ext deps can build
+RUN apk add git
+
+COPY . /tmp/blobstore
+RUN cd /tmp/blobstore \
+    && go build app/blobstore.go
+
+FROM alpine:3.9.4
+
+# These ARGs values are passed in via the docker build command
+ARG BUILD_DATE
+ARG VCS_REF
+ARG BRANCH=develop
+
+# Dockerize installation
+RUN apk add wget ca-certificates
+ENV DOCKERIZE_VERSION v0.6.1
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
```

Comment:
Don't use the jwilder binary, it doesn't include our site specific features. For example, none of the dockerize changes you or keith have requested are in there. Use the KBase version:
https://github.com/kbase/dockerize

Reply:
Huh, I just copied from @jayrbolton 's cache service. Jay FYI

Reply:
done